### PR TITLE
Manage Users UI Fixes + Basic Admin Edit & Delete with Dropdown

### PR DESCRIPTION
**Pull request by hadicheaito1**
## Description

- Front end changes to make consistent with other tables in admin panel
- 3 dots is a drop-down that allows the user to go to delete or edit user

## Screenshots (if appropriate):
image
image
image

### CODE REVIEW

**Comment by scouillard** [comment]
image

We lost the toggle arrow on the Navbar

File: app/assets/stylesheets/admin_panel.scss [hunk 1]
```diff
@@ -36,3 +36,7 @@
     }
   }
 }
+// Removes arrow from drop down
+.dropdown-toggle::after {
+  display: none !important;
```
Comment (scouillard) [inline app/assets/stylesheets/admin_panel.scss:41]: This needs to be scoped

**Comment by scouillard** [comment]
Didn't we say that clicking on the three dots would link to a dedicated "Edit User" page - and not be a dropdown?

**Comment by hadicheaito1** [comment]
Samuel and I discussed it, we agreed there is no need for drop-down since we can put Delete and Edit on the same page

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**